models/exchange/coinbase_pro/api.py

```diff
@@ -320,7 +320,7 @@ def limitSell(self, market: str='', base_quantity: float=0, futurePrice: float=0
         if not isinstance(base_quantity, int) and not isinstance(base_quantity, float):
             raise TypeError('The crypto amount is not numeric.')
 
-        if not isinstance(base_quantity, int) and not isinstance(base_quantity, float):
+        if not isinstance(futurePrice, int) and not isinstance(futurePrice, float):
```

Comment:
Can you change "futurePrice" too "future_price" please? Thanks.

Reply:
@dcolley, are you going to make this change?

Reply:
yes, I'll make the change and revert

Reply:
@dcolley, thanks. I'll merge it when is see your update.